bot.py

```diff
@@ -169,6 +170,23 @@ async def set_leaderboard_channel(ctx, channel: discord.TextChannel):
     main_leaderboard = Leaderboard(db_handler=db_handler, channel=channel)
     await ctx.respond("Leaderboard channel updated.")
 
+# Set Soloque Leaderboard Channel
+@commands.has_role("Staff")
+@bot.slash_command(description="Staff only command. Sets the leaderboard output channel.")
+async def set_soloque_channel(ctx, channel: discord.TextChannel):
+    if channel == None:
+        await ctx.respond("Channel not found. Send as a #channel.")
+        return
+    global solo_queue_leaderboard
+    if solo_queue_leaderboard != None:
+        solo_queue_leaderboard.channel = channel
+        await ctx.respond("Soloqueue Channel reset.")
+        return
+    solo_queue_leaderboard = Soloqueue_Leaderboard(db_handler=db_handler, channel=channel, region=my_region)
+    await ctx.respond("Set channel and set timer for soloqueue leaderboard")
```

Comment:
nit: I would change this to "Channel and timer set for soloqueue leaderboard", this current phrasing makes it sound like I still have to do it.